src/libraries/System.Resources.ResourceManager/System.Resources.ResourceManager.sln

```diff
@@ -1,4 +1,8 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
```

Comment:
I don't think we want these versions burned into the .sln files. Although, I'm not sure how to update them correctly...

Reply:
This seems like a losing battle since VS adds it anytime you make a change and save.

Reply:
@ViktorHofer - thoughts? Is there a well-defined process to update the .sln files?

Reply:
Please revert the solution file changes.

> This seems like a losing battle since VS adds it anytime you make a change and save. #Resolved

This is only true for solution files that contain System.Private.CoreLib's shared project file. Locally I'm experimenting with getting rid of that shared project file for CoreLib so that we can build it easier and include it as part of libraries which would solve this issue but I haven't anything to commit yet and would need to discuss the changes with a larger audience first.

> Is there a well-defined process to update the .sln files?

Yes there is. Do a `dotnet build src/libraires/slngen.proj` from a VS Developer Command Prompt which will update all slns. For the sake of this PR I would just revert the relevant changes.